app/controllers/YourSelfAssessmentChargesController.scala

```diff
@@ -90,9 +90,7 @@ class YourSelfAssessmentChargesController @Inject()(val authActions: AuthActions
     if (isEnabled(AdjustPaymentsOnAccount)) {
       claimToAdjustService.getPoaTaxYearForEntryPoint(nino).flatMap {
         case Right(value) => Future.successful(WYOClaimToAdjustViewModel(isEnabled(AdjustPaymentsOnAccount), value))
-        case Left(ex: Throwable) =>
-          Logger("application").error(s"Unable to create WYOClaimToAdjustViewModel: ${ex.getMessage} - ${ex.getCause}")
-          Future.failed(ex)
+        case Left(ex: Throwable) => Future.failed(ex)
```

Comment:
not sure why we drop logging here?

Reply:
think this was a merge issue with Deng's homepage PR, have added back now